docs/shared-content/installation/install-radius/initialize-radius.md

```diff
@@ -12,6 +12,17 @@ Initializing Radius installs the Radius control-plane and creates a Radius Envir
    kubectl config current-context
    ```
 
+1. Create a Kubernetes cluster
+
+   Radius runs inside [Kubernetes]({{< ref "guides/operations/kubernetes" >}}). Create one from the [supported k8s clusters]({{< ref "/guides/operations/kubernetes/overview#supported-kubernetes-clusters" >}}) 
```

Comment:
@Reshrahim Hey it seems you've duplicated the step above again by mistake.

Reply:
thanks for catching that.